transactionaldriver-and-tomcat/run.sh

```diff
@@ -0,0 +1,21 @@
+wget -nc http://mirror.vorboss.net/apache/tomcat/tomcat-7/v7.0.78/bin/apache-tomcat-7.0.78.zip
```

Comment:
A lot can fail here so you should bail out on error by adding the following to the top of the script:
```sh
set -e
```